Description: Fixing the threshold syntax for snmp-load check command
Forwarded: https://dev.icinga.org/issues/7275
Author: Jan Wagner <waja@cyconet.org>
Last-Update: 2014-09-18

From cd7248a74b66b32d0c239feca8efb61d679072e9 Mon Sep 17 00:00:00 2001
From: Jan Wagner <waja@cyconet.org>
Date: Thu, 18 Sep 2014 01:12:58 +0200
Subject: [PATCH] Fix '3 warnings and critical ' error

---
 itl/command-plugins-manubulon.conf | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/itl/command-plugins-manubulon.conf b/itl/command-plugins-manubulon.conf
index 910bf61..62a848a 100644
--- a/itl/command-plugins-manubulon.conf
+++ b/itl/command-plugins-manubulon.conf
@@ -85,8 +85,8 @@ object CheckCommand "snmp-load" {
 	}
 
 	vars.snmp_load_type = "stand"
-	vars.snmp_warn = 85
-	vars.snmp_crit = 95
+	vars.snmp_warn = "4,3,3"
+	vars.snmp_crit = "8,5,5"
 	vars.snmp_perf = true
 }
 
